Multi-assignment

dhwthompson opened the pull request:
Two judges per submission, as agreed.

dhwthompson committed:
Remove single-assignment constraint.

Unfortunately, due to a problem with South and InnoDB not wanting to
play together, the sanest way to do this is to drop the model entirely
and reinstate it in the new form. Yeah, I know it's a bit screwed up.

dhwthompson committed:
Update judge assignment with doubled-up judges.

rossbruniges commented on apps/challenges/tests/test_assignment.py:
```diff
@@ -103,7 +103,7 @@ def test_assignment_commit(self):
         self.assertEvenAssignment(assignments)
         self.assertEqual(JudgeAssignment.objects.count(),
-                         Submission.objects.count())
+                         Submission.objects.count() * 2)
```
If we change the number of judges this test will break. Is it possible to have this number imported from settings.py or from assign.py itself?

dhwthompson committed:
Pull number of judges per entry out into settings.

Also tweak the assignment test to use the setting instead of a
hard-coded multiplier.
